docs/common/rule.ldoc

```diff
@@ -50,6 +50,24 @@
 -- @see rule
 -- @see except
 
+--- A table which content will be compared to the target object current properties.
+--
+-- The comparison will be made using the lesser (`<`) operator.
+--
+-- @rulecomponent rule_lesser
+-- @param table
+-- @see rule
+-- @see except
+
+--- A table which content will be compared to the target object current properties.
+--
+-- The comparison will be made using the lesser (`>`) operator.
```

Comment:
Shouldn't this be `The comparison will be made using the greater (`>`) operator`?

Reply:
thanks, done